src/components/TimeseriesDataExport/TimeseriesDataExport.tsx

```diff
@@ -28,7 +37,10 @@ export type FetchCSVCall = (
   opts: CsvParseOptions
 ) => Promise<string>;
 
-export type FetchTimeseriesCall = (ids: InternalId[]) => Promise<TimeSeries[]>;
+export type FetchTimeseriesCall = (
+  ids: InternalId[]
+) => Promise<GetTimeSeriesMetadataDTO[]>;
+export type LabelFormatterForCSV = LabelFormatter;
```

Comment:
```suggestion
export type CSVLabelFormatter = LabelFormatter;
```
maybe?

Reply:
nope. `CSVLabelFormatter` –  more complex internal interface and here I'm only exporting function type